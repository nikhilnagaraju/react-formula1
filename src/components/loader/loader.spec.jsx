import { shallow } from 'enzyme';
import React from 'react';
import { Loader } from './';

describe('Image component', () => {
  it('snapshot matches', () => {
    const wrapper = shallow(
      <Loader>Lorem ipsum...</Loader>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
