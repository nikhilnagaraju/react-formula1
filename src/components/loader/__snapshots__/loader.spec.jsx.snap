// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component snapshot matches 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loader>
    Lorem ipsum...
  </Loader>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Loading...
        </h1>,
        <span
          className="spinner"
        />,
        <p>
          Lorem ipsum...
        </p>,
      ],
      "className": "loader",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Loading...",
        },
        "ref": null,
        "rendered": "Loading...",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "spinner",
        },
        "ref": null,
        "rendered": null,
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Lorem ipsum...",
        },
        "ref": null,
        "rendered": "Lorem ipsum...",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Loading...
          </h1>,
          <span
            className="spinner"
          />,
          <p>
            Lorem ipsum...
          </p>,
        ],
        "className": "loader",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Loading...",
          },
          "ref": null,
          "rendered": "Loading...",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "spinner",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Lorem ipsum...",
          },
          "ref": null,
          "rendered": "Lorem ipsum...",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
